### PR TITLE
Replace helm stable with BAC in chart readme

### PR DESCRIPTION
**Pull request by antgamdia**
### Description of the change

Minor PR replacing an old reference to the Helm stable repo with the Bitnami Application Catalog.

### Benefits

Up to date docs.

### Possible drawbacks

N/A, but CI will complain anyway :(

### Applicable issues

N/A

### Additional information

N/A

### CODE REVIEW

File: dashboard/package.json [hunk 1]
```diff
@@ -2,6 +2,7 @@
   "name": "dashboard",
   "version": "0.1.0",
   "private": true,
+  "proxy": "https://localhost",
```
Comment (absoludity) [inline dashboard/package.json:5]: Why this change?

**Review by absoludity — APPROVED** [review]
Unsure about the package.json change, but +1 for the doc fix.

File: dashboard/package.json (hunk 1 above)
Comment (antgamdia) [reply dashboard/package.json:5]: Oops, committed by mistake. It is what I use for local UI development :facepalm: 
Deleting the change straight away :S
